cicd/forgeops-tests/tests/smoke/idm/test_milestone_validation.py

```diff
@@ -0,0 +1,74 @@
+# Lib imports
+from requests import get, post, put, delete, session
+from kubernetes import client, config, stream
+from kubernetes.client import Configuration
+from kubernetes.client.rest import ApiException
+from kubernetes.stream import stream
+from pprint import pprint
+
+# Framework imports
+# from ProductConfig import IDMConfig
+# from utils import logger, rest
+
+
+class TestMilestoneValidation(object):
```

Comment:
I am unsure of the purpose of this class as there is an overlap with the metadata tests.
Also, the tests do not assert anything.
The file needs docstrings.